src/domains/elt/_modules/function_apps/function_app_elt.tf

```diff
@@ -105,6 +105,11 @@ locals {
       #
 
       INTERNAL_TEST_FISCAL_CODES = module.tests.users.all
+
+      # REDIS CACHE CONFIG FOR PDV IDs
+      REDIS_URL      = data.azurerm_redis_cache.ioauth_redis_common_itn.hostname
+      REDIS_PORT     = data.azurerm_redis_cache.ioauth_redis_common_itn.ssl_port
+      REDIS_PASSWORD = data.azurerm_redis_cache.ioauth_redis_common_itn.primary_access_key
```

Comment:
We should avoid using `PASSWORD` as the authentication method, it is preferable to use [Redis access policies ](https://github.com/pagopa/io-fims/blob/main/infra/resources/_modules/web_apps/linux_web_app_op.tf#L60) instead.

Reply:
than you @lucacavallaro. I would like to address this improvement in another PR